source/core/handling_msg.py

```diff
@@ -21,8 +21,8 @@
 
 
 def echo(text: str):
-    """On simple messages bot replies with echo"""
-    return text + '!'
+    """On simple messages bot replies that didn't understand user"""
+    return f'I didn\'t understand what you want.\nYou wrote: {text}'
```

Comment:
It is weird that bot tells what user wrote, because user can see what he wrote by checking the previous message